mmhuman3d/core/cameras/camera_parameter.py

```diff
@@ -343,6 +384,70 @@ def get_vibe_dict(self) -> dict:
         }
         return ret_dict
 
+    def check_item(self, key: Any, val: Any) -> None:
+        """Check whether the key and its value matches definition in
+        CameraParameter.SUPPORTED_KEYS.
+
+        Args:
+            key (Any):
+                Key in CameraParameter.
+            val (Any):
+                Value to the key.
+
+        Raises:
+            KeyError:
+                key cannot be found in
+                CameraParameter.SUPPORTED_KEYS.
+            TypeError:
+                Value's type doesn't match definition.
+        """
+        self.__check_key__(key)
+        self.__check_value_type__(key, val)
+
+    def __check_key__(self, key: Any) -> None:
+        """Check whether the key matches definition in
+        CameraParameter.SUPPORTED_KEYS.
+
+        Args:
+            key (Any):
+                Key in CameraParameter.
+
+        Raises:
+            KeyError:
+                key cannot be found in
+                CameraParameter.SUPPORTED_KEYS.
+        """
+        check_passed = True
+        if key not in self.__class__.SUPPORTED_KEYS:
+            check_passed = False
+        if not check_passed:
+            err_msg = 'Key check failed in CameraParameter:\n'
+            err_msg += f'key={str(key)}\n'
+            raise KeyError(err_msg)
+
+    def __check_value_type__(self, key: Any, val: Any) -> None:
+        """Check whether the type of value matches definition in
+        CameraParameter.SUPPORTED_KEYS.
+
+        Args:
+            key (Any):
+                Key in CameraParameter.
+            val (Any):
+                Value to the key.
+
+        Raises:
+            TypeError:
+                Value is supported but doesn't match definition.
+        """
+        check_passed = True
+        if type(val) != self.__class__.SUPPORTED_KEYS[key]['type']:
+            check_passed = False
+        if not check_passed:
+            err_msg = 'Type check failed in CameraParameter:\n'
+            err_msg += f'key={str(key)}\n'
+            err_msg += f'type(val)={type(val)}\n'
+            raise TypeError(err_msg)
```

Comment:
```
        if type(val) != self.__class__.SUPPORTED_KEYS[key]['type']:
            err_msg = 'Type check failed in CameraParameter:\n'
            err_msg += f'key={str(key)}\n'
            err_msg += f'type(val)={type(val)}\n'
            raise TypeError(err_msg)
```
same as above, `check_passed` doesn't seem to be returned or used downstream

Reply:
Fixed in 96daaac.